content/blog/spring-cleaning/index.Rmd

```diff
@@ -0,0 +1,84 @@
+
+The tidyverse team at Posit has a practice of tackling Spring Cleaning together  - we set aside a week every year to work in a semi-structured way to efficiently take care of a common list of package maintenance tasks. We find that setting a time for them and doing them all together during one week is an effective, and more fun, way to get them done. We recently completed our 2023 Spring Cleaning and thought it might be fun to share our process.
+
+## Preparation
+
+Early in the new year, we set aside the time in our calendars for Spring Cleaning - this way everyone knows that it's coming up and can make sure they have cleared the space in their schedules (and their minds) to focus on it.
+
+We prepare for the week by creating a list of things we want to take care of in our packages. Rather than adding features or fixing bugs, these tasks are usually about bringing things up to current standards or best practices, and include things like updating tests to the latest testthat version, updating pkgdown templates, and adding alt-text to images in pkgdown sites. Not surprisingly, this year a lot of the upkeep was related to our recent rebrand from RStudio to Posit -- things like updating the copyright holder and author email addresses, and using updated logos without the old rstudio.com website on them.
+
+We start off the week with a kickoff meeting on Monday morning. We go through the checklist with everybody and refine what's in it, making sure everybody has had input. Because we maintain so many packages, we have a spreadsheet where we keep track of the packages that are undergoing spring cleaning, and people can assign themselves to packages and mark them as completed when they're done.
+
+## Checklists, checklists, checklists
+
+We formalize these tasks into a checklist ([who doesn't love checklists](https://atulgawande.com/book/the-checklist-manifesto/)) via a function in usethis called `use_tidy_upkeep_issue()`. If you're a package developer and you use `use_release_issue()`, this will look familiar to you - it opens an issue in the package's GitHub repository with a checklist of tasks to guide you through what needs to be done. We update the function with the current year's checklist just prior to starting (and sometimes during) Spring Cleaning.
+
+Package maintainers then install the development version of usethis to get the current checklist, and call `usethis::use_tidy_upkeep_issue()` in their package to create the issue. If there are any tasks that aren't relevant to that particular repo it's easy to just edit the issue and remove it. To be really meta, here is the 2023 Spring Cleaning [upkeep issue for usethis](https://github.com/r-lib/usethis/issues/1791), created by usethis:
+
+```{r}
+#| echo: false
+#| fig-alt: Screenshot of an example upkeep issue in the usethis repository
```

Comment:
Should include any pertinent text from the screenshot in the alt tag.

Reply:
I've made the image itself a link, which I believe obviates the need for really descriptive alt-text, since the image is just a preview of the destination URL.